File: no1/php/b.php
<?php

class Queue {

  public function __construct($list){
    require_once("./a.php");

    $this->stack1 = new Stack($list);
    $this->stack2 = new Stack($list);
  }

  public function add($value){
    $this->stack1->T_push($value);
    $this->stack2->T_push($value);
  }

  public function T_peek(){
    $this->stack1->T_peek();
  }

  public function T_remove(){
    $this->stack1->T_remove();
  }

  public function size(){
    return $this->stack1->size();
  }
}

?>

File: no1/python/b.py
# !/usr/bin/env python
# -*- coding: utf-8 -*-

from a import Stack

class Queue:

    def __init__(self,stack):
        self.stack1 = Stack(stack)
        self.stack2 = Stack(stack)
        while !self.stack1.empty(): self.stack1.T_pop()
        while !self.stack2.empty(): self.stack1.T_push(self.stack2.T_pop())

    def add(self,obj):
        stac1.queue.append(obj)

    def T_peek(self):
        return self.stack1.T_peek()

    def T_remove(self):
        self.stack1.T_remove

    def size(self):
        self.stack1.size()
